exercises/binary-search/binary_search_test.py

```diff
@@ -2,6 +2,7 @@
 
 from binary_search import binary_search
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
```

Comment:
Looks like the version number is wrong - the canonical data is version `1.0.0`.

This like also need to have an extra blank line before it in order to pass the `flake8` tests for compliance with PEP8.

Reply:
Hi, so if I change nuber of version , all will pass good?